src/seedu/addressbook/AddressBook.java

```diff
@@ -1144,14 +1184,14 @@ private static String getUsageInfoForExitCommand() {
      */
 
     /**
-     * Removes sign(p/, d/, etc) from parameter string
+     * Removes prefix(p/, d/, etc) from parameter string
```

Comment:
this is still the same as original behavior, no? you are still talking in terms of address book flags.